### PR TITLE
Version 0.13.0

### PR DESCRIPTION
**Pull request by antalszava**
Changes the version number to `0.13.0` and adds the following files:
* .gitignore
* CHANGELOG.md
* MANIFEST.in
* LICENSE

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Release 0.13.0
+
+Initial release.
+
+This release contains contributions from (in alphabetical order):
+
+Chase Roberts, Alain Delgado Gran, Theodor Isacsson, Josh Izaac, Nathan
+Killoran, Antal Száva.
```
Comment (josh146) [inline CHANGELOG.md:8]: 🔥

**Review by josh146 — APPROVED** [review]
Looks good to me @antalszava! Before merging this, however, I recommend:

* Running `make dist` and `make wheel`, and make sure everything works
* Checking that the created source zip and wheel contain all the required files.

Do we want the top-level `setup.py` to also package the `src` folder?

**Comment by antalszava** [comment]
Oh thanks, those are good points!

1. 
> Running make dist and make wheel, and make sure everything works

`make dist` created `PennyLane-Orquestra-0.13.0.tar.gz` that contains the following files (as per `MANIFEST.in`):

image

2.
`make wheel`: created the wheel and after installing it in a clean `conda` environment, the unit tests ran well.

3.
> Do we want the top-level setup.py to also package the src folder?

That shouldn't be necessary as the contents of the `src` and `steps` folders are not used by the PennyLane-Orquestra plugin. Those folders are downloaded by Orquestra every time a workflow uses a step defined in `steps`. So changing the code in there and pushing to the `main` branch of this repo would take effect immediately (without the need of a release).

**Comment by josh146** [comment]
Thanks Antal for checking, looks good to me

**Pull request merged**